.eslintrc.yaml

```diff
@@ -67,6 +67,7 @@ overrides:
         [2, { accessibility: 'no-public' }]
       '@typescript-eslint/no-require-imports': 0
       '@typescript-eslint/promise-function-async': 0
+      react/prop-types: 0
```

Comment:
For TS files we now don't declare prop-types (they are generated using the babel plugin). Therefore, we need to turn off this eslint rule.

Reply:
Does this apply to all only .tsx files or all files?

Reply:
Yes, this is specific to the ts files